Add warning to clarify utilization of environment variable MANGER_SSL (#135)
neuvector/docs: docs/02.deploying/01.production/03.details/03.details.md

* MANAGER_SSL
> (Optional) Manager by default uses and HTTPS/SSL connection. Set the value to “off” to use HTTP.

:::warning important

> This environment variable is only for standalone NeuVector product. If it is under Rancher SSO, please set it to "on" or not to set it. NeuVector will use HTTPS by default.

Reviewer: ```suggestion
This environment variable is only for the standalone NeuVector product. Please set the variable to "on" if using Rancher SSO or leave unset as NeuVector will use HTTPS by default.
```
